bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details-utils.ts

```diff
@@ -102,3 +105,23 @@ export function createDataPoints(evaluationHistory: Trace[]): IDataPoint[] {
     .sort((a, b) => a.yElement.localeCompare(b.yElement));
   return [...scores, ...sortedResults];
 }
+
+export function parseSloOfEvaluations(evaluationTraces: Trace[]): void {
+  const unparsedEvaluations = (trace: Trace): trace is Trace & { data: { evaluation: IEvaluationData } } =>
```

Comment:
This could be defined outside of the function, makes it also testable.

Reply:
More testable only if I also export it, but why should I export something that I isn't need outside?